lib/l10n/app_es.arb

```diff
@@ -0,0 +1,185 @@
+{
+    "@@locale": "en",
```

Comment:
Could you please change the locale. The rest looks good. Thank you.

Reply:
Done! Sorry